kuma/settings/common.py

```diff
@@ -468,7 +480,11 @@ def enable_candidate_languages():
 )
 AUTH_USER_MODEL = 'users.User'
 USER_AVATAR_PATH = 'uploads/avatars/'
-DEFAULT_AVATAR = STATIC_URL + 'img/avatar.png'
+DEFAULT_AVATAR = config(
+    'DEFAULT_AVATAR',
+    default=STATIC_URL + 'img/avatar.png'
```

Comment:
I think it should be `STATIC_URL + config(......)`
The environment variable should able to override the file name only. the link should be serve as STATIC_URL.